src/theme/mod.rs

```diff
@@ -361,7 +362,8 @@ impl render::SizeColours for Theme {
 #[cfg(unix)]
 impl render::UserColours for Theme {
     fn you(&self)           -> Style { self.ui.users.user_you }
-    fn someone_else(&self)  -> Style { self.ui.users.user_someone_else }
+    fn other(&self)  -> Style { self.ui.users.user_other }
```

Comment:
Just a small formatting nitpick ... this line should be indented as the others of the implementation.